sdk/search/azure-search/src/samples/java/com/azure/search/CreateSkillsetExample.java

```diff
@@ -0,0 +1,117 @@
+        SearchServiceClient searchServiceClient = new SearchServiceClientBuilder()
+            .endpoint(ENDPOINT)
+            .credential(new SearchApiKeyCredential(ADMIN_KEY))
+            .buildClient();
+
+        createOcrSkillset(searchServiceClient);
+        createCustomSkillset(searchServiceClient);
+    }
+
+    private static void createOcrSkillset(SearchServiceClient searchServiceClient) {
+        List<InputFieldMappingEntry> inputs = Arrays.asList(
+            new InputFieldMappingEntry()
+                .setName("url")
+                .setSource("/document/url"),
+            new InputFieldMappingEntry().setName("queryString")
+                .setSource("/document/queryString")
+        );
+
+        List<OutputFieldMappingEntry> outputs = Collections.singletonList(
+            new OutputFieldMappingEntry()
+                .setName("text")
+                .setTargetName("mytext")
+        );
+
+        List<Skill> skills = Collections.singletonList(
+            new OcrSkill()
+                .setShouldDetectOrientation(true)
+                .setName("myocr")
+                .setDescription("Tested OCR skill")
+                .setContext("/document")
+                .setInputs(inputs)
+                .setOutputs(outputs)
+        );
+
+        Skillset skillset = new Skillset()
+            .setName("ocr-skillset")
+            .setDescription("Skillset for testing default configuration")
+            .setSkills(skills);
+
+        System.out.println(String.format("Creating OCR skillset '%s'", skillset.getName()));
+
+        Skillset createdSkillset = searchServiceClient.createSkillset(skillset);
+
+        System.out.println("Created OCR skillset");
+        System.out.println(String.format("Name: %s", createdSkillset.getName()));
+        System.out.println(String.format("ETag: %s", createdSkillset.getETag()));
+
+        System.out.println("\n");
+    }
+
+    private static void createCustomSkillset(SearchServiceClient searchServiceClient) {
+        HashMap<String, String> headers = new HashMap<>();
+        headers.put("Ocp-Apim-Subscription-Key", "foobar");
+
+        List<InputFieldMappingEntry> inputs = Collections.singletonList(
+            new InputFieldMappingEntry()
+                .setName("text")
+                .setSource("/document/mytext")
+        );
+
+        List<OutputFieldMappingEntry> outputs = Collections.singletonList(
+            new OutputFieldMappingEntry()
+                .setName("textItems")
+                .setTargetName("myTextItems")
+        );
+
+        Skill webApiSkill = new WebApiSkill()
+            .setUri("https://example.com")
+            .setHttpMethod("POST") // Supports only "POST" and "PUT" HTTP methods
+            .setHttpHeaders(headers)
+            .setInputs(inputs)
+            .setOutputs(outputs)
+            .setName("webapi-skill")
+            .setDescription("A WebApi skill that can be used as a custom skillset");
```

Comment:
We should have a better description here "A WebApiSkill that can be used to call a custom web api function.".  "Custom skillset" as a concept doesn't really make sense.

Reply:
Please advice on a different description

Reply:
I had given one:

"A WebApiSkill that can be used to call a custom web api function."

Reply:
or maybe like "A WebApiSkill that can be used to send data to a custom, external HTTP endpoint for custom enrichment"?